packages/lib/services/rest/routes/notes.test.ts

```diff
@@ -0,0 +1,82 @@
+import Logger from '@joplin/utils/Logger';
+import shim from '../../../shim';
+import uuid from '../../../uuid';
+import { downloadMediaFile } from './notes';
+
+describe('routes/notes', () => {
+
+	beforeEach(() => {
+		jest.resetAllMocks();
+	});
+
+	test.each([
+		'/invalid/url',
+		'htp/asdfasf.com',
+		'https//joplinapp.org',
+	])('should not return a local file for invalid protocols', async (invalidUrl) => {
+
+		expect(downloadMediaFile(invalidUrl)).resolves.toBe('');
```

Comment:
```suggestion
		await expect(downloadMediaFile(invalidUrl)).resolves.toBe('');
```

This may need an `await`. See [docs](https://jestjs.io/docs/asynchronous#asyncawait).